src/sentry/static/sentry/app/views/discover/styles.tsx

```diff
@@ -33,23 +31,6 @@ export const DiscoverContainer = styled('div')`
   }
 `;
 
-export const DiscoverGlobalSelectionHeader = styled(GlobalSelectionHeader)`
-  position: absolute;
-  top: 0;
-  left: 0;
-`;
```

Comment:
This absolute positioning seems to be unnecessary. (For some reason this styled component was also giving me issues with ts while refactoring GlobalSelectionHeader, which is the main driving force here).

Reply:
Damn looks like we do need it